examples/component-scripts/zarf.yaml

```diff
@@ -4,14 +4,26 @@ metadata:
   description: "Test component to demonstrate script timeout feature"
 
 components:
+  - name: create
+    required: true
+    scripts:
+      # Note these files will be created in this directory, regardless of where the package create command is called from
+      create:
```

Comment:
do we need both before and after create functions? (might be weird to take actions after the .zst file is built?)